### PR TITLE
Disable Session Storage buffer experiment

### PR DESCRIPTION
**Pull request by kamilszostak**

### CODE REVIEW

**Comment by kamilszostak** [comment]
I've decided to disable SessionStorage buffer as it might be a root cause of #217 . 
I'll investigate and re-enable the new buffer when we have a fix.

File: package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-        "microsoft",
-        "azure"
-    ],
+    "keywords":  [
+                     "browser performance monitoring",
+                     "script errors",
+                     "application insights",
+                     "microsoft",
+                     "azure"
+                 ],
```
Comment (dnduffy) [inline package.json]: nitpick: is the formatting change intentional or did VS auto-format it?

**Comment by dnduffy** [comment]
:shipit:

File: package.json (hunk 1 above)
Comment (kamilszostak) [inline package.json]: Not intentional. Powershell script to update the version number did that. 
I'll check if I can fix it later today and submit a fix in a separate pull request.

**Comment by AlexBulankou** [comment]
:shipit:

**Comment by MaxShehovtsov** [comment]
:shipit:

**Pull request merged**